### ISSUE TITLE
Addition of an afterburner run

### ISSUE THREAD

**Issue by elfnerhannah** [smash-transport/smash-analysis#37]
I think it would be useful to include two smooth hypersurface files for 200 GeV AuAu at RHIC and 5.02 TeV PbPb at LHC to run a SMASH afterburner to ensure that nothing changes there. This is a use case of many external groups, therefore I think it would be beneficial to keep track of changes. 
@akschaefer and @thegarzo might have comments.

**Comment by elfnerhannah** [comment]
The file size is an important point to think about, what the best way of dealing with them is. For the physics-wise realistic initial conditions, this is not so important here, since it is just about documenting changes. Any IC would do for that purpose.

**Comment by thegarzo** [comment]
I don't think there is any way of getting out the size problem, apart from reducing the lattice size and volume of the hydro run. However, we know thanks to @akschaefer that below some threshold, it breaks conservation of energy, etc. I can of course try to engineer a FO surface that we can use, that is not so heavy. 
Wrt the physics, yes, I imagined, but I think that external people would get confused by such a "discrepancy" that is why I wanted to point it out.

**Comment by elfnerhannah** [comment]
Sure, if it is easy to produce an averaged Glauber IC that allows to match data roughly that would of course be nice. 
I would not attempt to reduce the size of the files, we just have to think about where to store them. They do not need to be part of the git repo for example.

**Comment by akschaefer** [comment]
I think this is in principle a good idea. Maybe it makes sense if we clarify where exactly we want to run this. As another runtest for the afterburner? As part of the smash-analysis? Or elsewhere during the tagging procedure?
Regarding the grid size for the hydro. I think it might not be too bad if we reduce the grid spacing at these high energies. The major problems with too large grid cells occurred at low collision energies. The higher we go in energy, the less of a problem it is. If you want I can run RHIC and LHC at differently fine grids to see how unfine we can go without major issues. In any case, I think it's doable to produce a file that is not extremely large.

**Comment by akschaefer** [comment]
Sorry, my bad. I just realized this is an analysis suite issue. So, I presume it's meant to be part of the smash-analysis in the future :)

**Comment by NGoetz** [comment]
Just to be clear, is this issue intended to add the hypersurface output files from Hydro runs, which are then mapped to particle lists by the sampler and given as an input to SMASH as afterburner inside the analysis suite, or only particle lists produced by the sampler? In the latter case, file size would be not that much of a problem.

**Comment by akschaefer** [comment]
You are absolutely right, @NGoetz. It makes no sense at all to include the hydro freeze out files as we need the intermediate sampler stage in order to produce particle lists for the SMASH afterburner 🤦‍♀️ . If we don't want to additionally test the sampler (which I would not do within the SMASH analysis, as it is meant as a test suite for SMASH and not any module of the hybrid), we need to prepare a particle list instead of the hydro hypersurface. And yes, you are right again. For this, file size is not an issue. We can easily reduce it to 50 or 100 events and we should be save. Thanks for making this clear.

**Comment by elfnerhannah** [comment]
I fully agree with @NGoetz. My main idea is to test that SMASH as an afterburner does not change drastically for new versions. (or at least we are aware of changes..). I would start with a basic check of bulk observables like spectra and v2 from kaons, pions, protons similar to what Anna did for the photon paper.
